### DISCUSSION TITLE
A new web app usign section-property module

### DISCUSSION THREAD

**Discussion by linokl** [robbievanleeuwen/section-properties#275] [Show and tell]
Hi all,

I have made a web app using the section property module. Please check it out and let me know your feedback. It doesn't use all the features of the module yet. It is the first release.

https://www.standardsolver.com/sec-property

Regards
Lino

**Comment by Czarified** [comment]
Lino,

This is a really neat website and use case for SectionProperties! Thanks for sharing!

Do you have repos or links to source code for the other tools? And for the sectionproperties page, I'd suggest being very clear as to where the calculations are coming from. For students this doesn't matter, but for any professional work the source code and/or methods must be very clear, preferably with citations.

**Reply by linokl** [reply]
Hi Benjamin,

Thanks for the feedback. There are no repos for the website. The website is made using Plotly Dash. It is just a user interface over some of the publically available Python modules for structural analysis. My aim is to make a structural design website the same way Figma or Canva for UX design.

I will make it clear which modules are used for computation.
Beam and frames are analyzed using PyNiteFEA (https://pypi.org/project/PyNiteFEA/)
I will try to make the calculation more transparent during future updates. Hope you find the site useful.

Regards
Lino

**Comment by robbievanleeuwen** [comment]
Looks great @linokl, thanks for showcasing! We definitely welcome anything that makes ``sectionproperties`` more accessible!

I might have missed it, but I would appreciate attribution as per the license requirements, especially as it looks like you are thinking of making it commercial.

**Comment by linokl** [comment]
Hi Robbie,

I will add a link to the GitHub as a reference in the next update is that what is needed?. I am not that much a software guy! The python module is really cool I will try to incorporate more capabilities as I go on. The plan for the website is to be free for most of the task. Probably model storage need patreon subscriptions for more than 5 models.

**Reply by robbievanleeuwen** [reply]
Sounds good, best of luck with it 😄
